core/src/main/java/io/parsingdata/metal/data/selection/ByName.java

```diff
@@ -52,15 +53,15 @@ public static ParseValue getValue(final ParseGraph graph, final String name) {
      * @param name Name of the value
      * @return All values with the provided name in this graph
      */
-    public static ImmutableList<ParseValue> getAllValues(final ParseGraph graph, final String name) {
+    public static ImmutableList<Value> getAllValues(final ParseGraph graph, final String name) {
```

Comment:
There is an issue with the current definition of `ImmutableList.wrap()` that doesn't allow it. We could investigate an alternative declaration for either the `wrap()` method or `ValueExpression.eval()`.